SharePointDsc/DSCResources/MSFT_SPProductUpdate/readme.md

```diff
@@ -19,6 +19,12 @@ stream is added to indicate that the file is potentially from an unsafe source.
 To use these files, make sure you first unblock them using Unblock-File.
 SPProductUpdate will throw an error when it detects the file is blocked.
 
+NOTE2:
+When specifying the ShutdownServices parameter, the resource is stopping
+several SharePoint services and pausing all search crawls. After patching
+is complete, the search crawls is only resumed when all servers in the farm
```

Comment:
Small correction, since crawls is plural, the sentence should read: "... the search crawls ARE only resumed when..."